activemq-karaf/src/main/resources/features-core.xml

```diff
@@ -63,4 +63,12 @@
       <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-databind/${jackson-databind-version}</bundle>
       <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson-version}</bundle>
     </feature>
+
+    <!-- JMS1.1 and JMS2.0 support via the same Geronimo JMS 2 bundle  -->
+    <!-- Ideal since all JMS code shares the same class definitions    -->
+    <!-- Fortunately, JMS2.0 is fully backward compatible with JMS 1.1 -->
+    <feature name="activemq-combined-jms-spec">
```

Comment:
I think we could have done this directly in the activemq-client feature, no need to separate.